app/screens/login/LoginScreen.js

```diff
@@ -59,23 +60,21 @@ class LoginScreen extends Component {
       console.log('UserName False')
     }
   }
-
+  _getUser() {
+    const { navigation } = this.props;
+    if (navigation.state.params) {
+      this.setState({
+        paramsLogin: {
+          userName: navigation.state.params.userName,
```

Comment:
chỗ này refactor code như sau:
const {paramsLogin} = this.state;
const {params} = this.props.navigation.state;

paramsLogin.userName = params.userName;
paramsLogin.userName = params.password;
this.setState({paramsLogin})